backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/after_migration_sync_translations.sql

```diff
@@ -9,7 +9,7 @@ $$
 DECLARE
     count_updated NUMERIC;
 BEGIN
-    SELECT 1 FROM update_TRL_Tables_On_AD_Element_TRL_Update(NULL, NULL);
+    PERFORM update_TRL_Tables_On_AD_Element_TRL_Update(NULL, NULL);
```

Comment:
@metas-rc  couldn't you call this function without any param?

Reply:
done